src/app/zap-templates/templates/app/CHIPClusters.zapt

```diff
@@ -14,22 +15,23 @@
 namespace chip {
 namespace Controller {
 
-{{#chip_client_clusters}}
-class DLL_EXPORT {{asUpperCamelCase name}}Cluster : public ClusterBase
+{{/first}}
+class DLL_EXPORT {{asUpperCamelCase clusterName}}Cluster : public ClusterBase
 {
 public:
-    {{asUpperCamelCase name}}Cluster() : ClusterBase(app::Clusters::{{asUpperCamelCase name}}::Id) {}
-    ~{{asUpperCamelCase name}}Cluster() {}
-    {{#chip_cluster_commands}}
+    {{asUpperCamelCase clusterName}}Cluster() : ClusterBase(app::Clusters::{{asUpperCamelCase clusterName}}::Id) {}
+    ~{{asUpperCamelCase clusterName}}Cluster() {}
+    {{#all_outgoing_commands_for_cluster clusterName clusterSide isMfgSpecific="false"}}
     {{#first}}
 
     // Cluster Commands
     {{/first}}
-    CHIP_ERROR {{asUpperCamelCase name}}(Callback::Cancelable * onSuccessCallback, Callback::Cancelable * onFailureCallback{{#chip_cluster_command_arguments_with_structs_expanded}}{{#if_is_struct type}}{{else}}, {{chipType}} {{asLowerCamelCase label}}{{/if_is_struct}}{{/chip_cluster_command_arguments_with_structs_expanded}});
-    {{/chip_cluster_commands}}
+    CHIP_ERROR {{asUpperCamelCase commandName}}(Callback::Cancelable * onSuccessCallback, Callback::Cancelable * onFailureCallback{{#zcl_command_arguments}}{{#if_is_struct type}}{{else}}, {{as_underlying_zcl_type type}} {{asLowerCamelCase label}}{{/if_is_struct}}{{/zcl_command_arguments}});
```

Comment:
I have not looked the source code for {{zcl_command_arguments}} but I suspect that something is different compare to {{chip_cluster_command_arguments_with_structs_expanded}) since it seems that the command arguments have disappeared for the "controller-clusters.zap" configuration.

Reply:
Once I changed the parent block helpers I could no longer use {{chip_cluster_command_arguments_with_structs_expanded}}. Would it be possible to remove the restrictions in chip_cluster_command_arguments_with_structs_expanded. It makes more sense to continue using this and not zcl_command_arguments. I also noticed that the zap helper files are now being created outside the zap repo, could we bring these back into the zap repo? I feel it makes more sense to have them in the zap repo.